### PR TITLE
Add eraser new

### LINKED ISSUE #20: Add eraser
Is your feature request related to a problem? Please describe.
No

Describe the solution you'd like
Be able to erase paintbrush drawing 

Describe alternatives you've considered
N/A

Additional context
N/A

### PR DESCRIPTION
**Pull request by cooper667**
# Description

Adds the functionality to erase paintbrush drawings on a layer.

How the feature is used:

The user clicks on the eraser button to activate the eraser feature. They move their icon over any paintbrush drawings to erase it. They can toggle between the eraser mode and paintbrush mode by clicking on their associated buttons.   

closes #20

# Dependency changes

N/A

# Testing

N/A

# Documentation  
N/A

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/toolboxes/paintbrush/Canvas.tsx [hunk 1]
```diff
@@ -34,56 +35,84 @@ export class PaintbrushCanvas extends Component<Props> {
...
+
+    if (this.props.brushType == "eraser") {
+      if (this.isDrawing) {
+        // Add new point
+        this.points.push({ x, y });
+
+        // Draw current points
+        this.drawPoints(
+          this.points,
+          "#0000FF",
```
Comment (ChasNelson1990) [inline src/toolboxes/paintbrush/Canvas.tsx:68]: What is this hardwired colour for? Does this mean the eraser colour is bright green?

**Review by ChasNelson1990 — APPROVED** [review]
Can we add a keybinding to toggle the eraser on and off? E.g. hold ctrl down when moving the mouse or something?

File: src/toolboxes/paintbrush/Canvas.tsx (hunk 1 above)
Comment (cooper667) [reply src/toolboxes/paintbrush/Canvas.tsx:68]: No, you don't see the eraser as it's a subtraction anyway

**Comment by cooper667** [comment]
> Can we add a keybinding to toggle the eraser on and off? E.g. hold ctrl down when moving the mouse or something?

We can do that in another PR?

**Comment by ChasNelson1990** [comment]
> > Can we add a keybinding to toggle the eraser on and off? E.g. hold ctrl down when moving the mouse or something?
> 
> We can do that in another PR? 

Ace

**Review by SilviaZeta — APPROVED** [review]

**Review by philipjackson — APPROVED** [review]

**Pull request merged**